trend_micro_vision_one_xdr/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "5efb1591-f9ef-45a3-8b8e-9f716df68f16",
+  "app_id": "trend-micro-vision-one-xdr",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Gain insights into trend micro vision one xdr logs",
+    "title": "Trend Micro Vision One XDR",
+    "media": [],
+    "classifier_tags": [
+      "Category::Log Collection",
+      "Category::Security",
+      "Offering::Integration",
+      "Submitted Data Type::Logs"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "auto_install": false,
+      "source_type_id": 22902543,
+      "source_type_name": "trend-micro-vision-one-xdr",
```

Comment:
This should just be the natural name of the integration: Trend Micro Vision One XDR

Reply:
Done